frontend/pages/policies/ManagePoliciesPage/components/AddPolicyModal/AddPolicyModal.tsx

```diff
@@ -91,23 +137,44 @@ const AddPolicyModal = ({
     );
   });
 
+  const renderNoResults: JSX.Element = (
+    <>
+      There are no results that match your filters.{" "}
+      <CustomLink
+        text="Everyone can contribute"
+        url={CONTRIBUTE_TO_POLICIES_DOCS_URL}
+        newTab
+      />
+    </>
+  );
+
   return (
     <Modal
       title="Add a policy"
       onExit={onCancel}
       className={baseClass}
-      width="xlarge"
+      width="large"
     >
       <>
-        <div className={`${baseClass}__create-policy`}>
+        <div className={`${baseClass}__description`}>
           Choose a policy template to get started or{" "}
           <Button variant="text-link" onClick={onCreateYourOwnPolicyClick}>
             create your own policy
           </Button>
           .
         </div>
+        <Dropdown
+          value={platform}
+          className={`${baseClass}__platform-dropdown`}
+          options={PLATFORM_FILTER_OPTIONS}
+          searchable={false}
+          onChange={onPlatformFilterChange}
+          tableFilterDropdown
+        />
         <div className={`${baseClass}__policy-selection`}>
-          {policiesAvailable}
+          {renderFilteredPolicies.length > 0
```

Comment:
related to above, `renderFilteredPolicies.length` looks like you are accessing the length of a funciton, which wouldn't make sense, though it actually does since this is an array not a function. Are you okay with renaming to `rendered`...? Or `mapped` or something.

Reply:
Ok, I looked at what I wrote for a similar modal with similar buttons, and I was just explicit with defining the count as a new variable `queriesCount` and rendered the jsx buttons as `{queryList}` so I updated the variables here for consistency to use `filteredPoliciesCount` and `{filteredPoliciesList}`.